alb.tf

```diff
@@ -36,14 +44,9 @@ resource "aws_alb_target_group" "app" {
 resource "aws_lb_listener_rule" "host_based_weighted_routing" {
   listener_arn = data.aws_alb_listener.front_end.arn
 
-  action {
+  default_action {
     type             = "forward"
     target_group_arn = aws_alb_target_group.app.arn
   }
 
-  condition {
-    host_header {
-      values = [for sn in var.service_names : "${sn}.*"]
-    }
-  }
 }
```

Comment:
We can remove the entire `resource "aws_lb_listener_rule" "host_based_weighted_routing"` it is covered by the default action. Sorry my bad

Reply:
to be clear it is handled by the default action on the listener